WordPress/wordcamp.org
# Add capability checks and admin UI for viewing/moderating feedback

* Adds custom meta capabilities for viewing and moderating feedback, and maps them.
* Adds an admin page with a list table for reviewing feedback comments that operates similarly to Core's **edit-comments.php**.
* Adds a column on the Sessions list table to show feedback counts.
* Adds tests.

This is kind of a big PR. I could try and split it up, but I'm not sure how best to do it so the pieces make sense on their own.

Fixes #339 Fixes #340 

**Screenshots**

![feedback_screen](https://user-images.githubusercontent.com/916023/77012910-28128000-692c-11ea-92da-78034bcfd072.jpg)

![sessions_screen](https://user-images.githubusercontent.com/916023/77012919-2ea0f780-692c-11ea-9e39-d82531a23c91.jpg)

**To test:**

1. Rebuild the assets to get the right styles.
1. Add some normal comments to a blog post or two.
1. Create some session posts and add some feedback comments to one or more of them. The testing instructions on #353 have more details of a way to do that.
1. View the Comments screen from the Admin side menu. No feedback comments should appear, or be included in the counts.
1. Click the Feedback submenu item under Sessions. This should show a list table of all of the feedback you just created, but no non-feedback comments.
1. Try clicking the linked column headings to change the sorting of the feedback. Particularly the Rating column.
1. Try using the row actions and bulk edit to change the status of some feedback comments. Check that the counts in the view links at the top update correctly.
1. Try using the search box to find comments with only a particular word or phrase in the feedback answers.
1. Go to the Sessions list table. Sessions that have feedback comments should show the counts for them in a new column. Clicking on one of the feedback bubbles should go to the Feedback list table, but only show it for the one particular session.
1. Log into the Admin with a user that has Author role or below. The Feedback list table screen should not be accessible.
